### PR TITLE
Vale "slash" rule reenabled. Config changes trigger workflow

### PR DESCRIPTION
**Pull request by apinnick**
Enabled "RedHat.Slash" rule and updated paths in GitHub workflow vale.yml file.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: .github/workflows/vale.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/**.adoc'
```
Comment (ekohl) [inline .github/workflows/vale.yml:6]: This means it won't trigger on `adoc` files anymore, only when `valie.yml` is changed. I don't think that's what you intended. Instead I think we should add other vale related files to this list.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: .github/workflows/vale.yml (hunk 1 above)
Comment (apinnick) [reply .github/workflows/vale.yml]: I added an error .adoc file after reverting this line, but Vale did not check the new file.

Comment (apinnick) [reply .github/workflows/vale.yml:6]: I see the problem. Vale is only going to report actual errors. I'm going to raise the minimum report level from "error" to "warning" to see how that goes.

Comment (apinnick) [reply .github/workflows/vale.yml:6]: Works now! "warning" level was not a success, so I reverted to "error" level reporting and made sure I had real errors.

File: .github/workflows/vale.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ on:
     paths:
       - 'guides/common/**.adoc'
       - '.github/workflows/vale.yml'
+      - '.vale/styles/config/**.txt'
```
Comment (ekohl) [inline .github/workflows/vale.yml:8]: How about just triggering on any vale change?
```suggestion
      - '.vale/**'
      - .vale.ini
```

Comment (ekohl) [reply .github/workflows/vale.yml:8]: But this is only when we trigger the job. IMHO anything in `.vale` affects how Vale behaves so we should trigger CI on it. There's no reason to be more specific. `.vale/**` should still trigger when `.vale/styles/config/vocabularies/Foreman/accept.txt` changes.

Comment (apinnick) [reply .github/workflows/vale.yml:8]: OK, you might have a point. I have never tested this. I will make the change and see what happens.

**Review by ekohl — COMMENTED** [review]
The workflow changes look good to me, but could you rebase? It looks like it includes some changes that we merged to master already.

**Review by maximiliankolb — APPROVED** [review]
if you squash: LGTM

**Comment by apinnick** [comment]
> if you squash: LGTM

Squashed!

**Review by ekohl — COMMENTED** [review]
I'd prefer a more descriptive commit message. https://cbea.ms/git-commit/ is IMHO still the best reference on the topic. If I'm pedantic, I'd even suggest two commits because each change is logically speaking separate but I don't think others enforce that.

**Comment by apinnick** [comment]
> I'd prefer a more descriptive commit message. https://cbea.ms/git-commit/ is IMHO still the best reference on the topic. If I'm pedantic, I'd even suggest two commits because each change is logically speaking separate but I don't think others enforce that.

I will update the commit message but I don't think breaking the squashed commit is necessary.

**Review by maximiliankolb — APPROVED** [review]

**Pull request merged**